documentation/book/proc-uninstalling.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assemblies:
+//
+// master.adoc
+
+[id='ininstalling-{context}']
+= Uninstalling {ProductName}
+
+. Identify resources:
++
+--
+* Referenced by `Kafka`, `KafkaConnect` or `KafkaMirrorMaker`
```

Comment:
I have changed this to a prereq and switched the content so we're more specific about the type of thing that needs to be identified. @tombentley - is there any way we can make it more explicit exactly what the user needs to identify?

Reply:
It's a bit difficult because precisely which resources need deleting by the user depends on whether they're deleting `Kafka`, `KafkaConnect` or `KafkaMirrorMaker`. For example:

* For the logging config maps, it's only the case of logging configured via "external" ConfigMaps (i.e. when `type == external`) which would need manually deleting. 
* The CA secrets are only for `Kafka` and only when `Kafka.spec.clusterCa.generateCertificateAuthority` and/or `Kafka.spec.clientsCa.generateCertificateAuthority` is false. 
* Both `KafkaConnect` and `KafkaMirrorMaker` support a `template.pod.imagePullSecrets` etc.

If _really_ necessary we could probably come up with a list of paths for each resource kind which the user could check, but maintaining that would be quite onerous, so I'd rather avoid having to do that if we can find some suitable wording. The principle we're using is: If the user had to explicitly create the resource and only _referenced_ it from their Strimzi resource then it's their responsibility to delete it.

Reply:
Thanks Tom. Let's try and avoid the possible maintenance overhead of a list then. Here are the prereqs. If they're okay, I'll merge the changes. 

**Prerequisites**

In order to perform this procedure, identify resources created specifically for a deployment and referenced from the {ProductName} resource. 

Resources might include:

- Secrets (Custom CAs and certificates, Kafka Connect secrets, and other Kafka secrets)
- Logging `ConfigMaps` (of type `external`)

These are resources:

- Referenced by `Kafka`, `KafkaConnect`, `KafkaConnectS2I`, or `KafkaMirrorMaker` configuration
- Consumed by the operators